src/app/backend/resource/deployment/util/deploymentutil.go

```diff
@@ -0,0 +1,100 @@
+package util
```

Comment:
Ye, It would be great if they could move utils also. Keeping deployment utils deps also downloads significant amount of files so I guess it's better for now to just copy these few functions and keep them in sync.